text/README.md

```diff
@@ -24,6 +25,24 @@ before it is stored in a `text`-field along with the
 `uuid`, `meta_location`, and `data_location`.
 
 ## Usage
+Note that `&start=0&size=10` can be added
```

Comment:
Tiny semantic remark: Pagination parameters are usually named «offset» (start) and «limit» (size).